opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -58,6 +58,7 @@ class DeckKeyword;
 class UDQActive;
 class UDQConfig;
 class SICD;
+class AutoICD;
```

Comment:
I am attempted to change AutoICD to AICD, since we changed SpiralICD to SICD. :)

Reply:
Feel free!

Reply:
Let us put renaming in a separate PR, since that should not affect the results at all.